arch/configure.defaults

```diff
@@ -2109,9 +2109,51 @@ AR              =      ar
 ARFLAGS         =      ru
 M4              =      m4 -B 14000
 RANLIB          =      ranlib
-RLFLAGS         =
+RLFLAGS               =
```

Comment:
@weiwangncar Was the introduction of this whitespace change intentional?

Reply:
@mgduda Definitely not intentional. It was cut and pasted. Let's see if it can be removed.

Reply:
Looks good now. Thanks!